proto/asn1-pdu/asn1_universal_types.proto

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+// This proto represents the Universal types found in X509 Certifiactes.
+syntax = "proto2";
+
+import "google/protobuf/timestamp.proto";
+
+package asn1_universal_types;
+
+// This represents an ASN.1 BITSTRING, which denotes an
```

Comment:
```suggestion
// This represents an ASN.1 BIT STRING, which denotes an
```

`BitString` is the type name (X.680 (2015), 22.1), which in the schema is represented by `BIT STRING`. Since you opted for the all-cap name for integer, and to keep it easy with the time types and future expansions, let's just add it by its schema-identified name :)